CHANGELOG

```diff
@@ -2,6 +2,7 @@ Unreleased: mitmproxy next
 
 ** Full Changelog **
 
+    * UI appears blank in mitmweb for mitm.it flow(@samhita-sopho)
```

Comment:
Please rephrase to something "that you fixed".
"fixed xyz" or "added feature xyz"

Reply:
Thank you for pointing it , after updating the same I found that the test for web ui fails so working on it